app/src/androidTest/java/com/cliqz/extension/CliqzExtensionTest.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.cliqz.extension
+
+import androidx.test.platform.app.InstrumentationRegistry
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.mozilla.reference.browser.components.Core
+
+class CliqzExtensionTest {
+
+    lateinit var core: Core
+
+    @Before
+    fun setUp() {
+        val context = InstrumentationRegistry.getInstrumentation().context
+        core = Core(context)
+    }
+
+    @After
+    fun tearDown() {
+        core.cliqz.runtime.shutdown()
+    }
+
+    @ExperimentalCoroutinesApi
```

Comment:
If all the test use `@ExperimentalCoroutinesApi` why didn't you annotate the entire class?